_posts/2020-04-15-Jitsi-Instance-on-meet-opensuse-org.md

```diff
@@ -0,0 +1,41 @@
+---
+author: Marco Strigl 
+comments: true
+date: 2020-04-15 08:00:08+00:00
+layout: post
+title: Jitsi instance on meet.opensuse.org
+categories:
+- Announcements
+- Heroes 
+- Community
+tags:
+- video 
+- stayathome
+- gettogether
+- opensuse
+---
+
+In the times of Covid-19 and the people staying at home it is an adventure to get the tools to work from home without missing the benefits of face-to-face meetings.
+
+There are a lot of solutions out in the wild. But one promising solution is [Jitsi](https://jitsi.org/). Until now we used the instances provided by other people.
+
+But now we are able to introduce:
+
+[Our own Jitsi instance](https://meet.opensuse.org)
+
+It is based on openSUSE Leap 15.1 and uses docker containers to deploy Jitsi. The current security warnings were also considered, furthermore the setup uses secure LDAP and HTTPS.
```

Comment:
Any chance to build that properly instead of deploying some container aka random binary blob?

Reply:
I don't think this is the right place for technical inquiries. Maybe there's some project on progress-o-o?

Reply:
Yup, please send the suggestion to admin@opensuse.org, it will be considered :P

Reply:
That will be the next step. I also want to package jitsi for openSUSE. 
I will mention this in my post